src/ledger/ledger-db/k8s/overlays/development/kustomization.yaml

```diff
@@ -14,8 +14,11 @@
 
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+commonLabels:
+  tier: db
 resources:
   - ../../base
   - ledger-db.yaml
 components:
-  - ../../../../../components/development
\ No newline at end of file
+  - ../../../components/ledger
```

Comment:
Should this be one more directory up?

`../../../components` gets you here: `src/ledger/ledger-db/k8s/components/` and there's no `ledger` in there.